examples/v1/taskruns/sidecar-ready.yaml

```diff
@@ -18,6 +18,15 @@ spec:
           - -c
           - sleep 5 && touch /shared/ready
         timeoutSeconds: 10
+      # Adding startup probe for k8s native sidecar support
```

Comment:
@kgcarr we may want to document this change in https://github.com/tektoncd/pipeline/blob/main/docs/tasks.md#specifying-sidecars

1. An additional feature flag
2. Using a startup probe instead of a readiness probe